### DISCUSSION TITLE
Best way to implment Attribute Based Access Control

### DISCUSSION THREAD

**Discussion by NicholasMKim** [prisma/prisma#12648] [Q&A]
Hi team, I'm learning Prisma's good practices. 
What would be the best practice for implementing ABAC? e.g. `user`s can access the `post`s with matching `authorId`

**Comment by nurul3101** [comment]
Hey @mkim871 👋 
[Cerbos](https://cerbos.dev/ecosystem/cerbos-prisma) might be useful to you to implement authorisation.

They have an integration with Prisma and provide granular level access control which you are trying to achieve.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
